webapp/components/notifications/Notification/Notification.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+  <ds-space :class="[{ read: notification.read }, notification]" margin-bottom="x-small">
+    <no-ssr>
+      <ds-space margin-bottom="x-small">
+        <hc-user
+          v-if="resourceType == 'Post'"
+          :user="post.author"
+          :date-time="post.createdAt"
+          :trunc="35"
+        />
+        <hc-user v-else :user="comment.author" :date-time="comment.createdAt" :trunc="35" />
+      </ds-space>
+      <ds-text color="soft">
```

Comment:
> [<img alt="roschaefer" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/roschaefer) **Authored by [roschaefer](https://github.com/roschaefer)**
_<time datetime="2019-08-15T23:12:16Z" title="Friday, August 16th 2019, 1:12:16 am +02:00">Aug 16, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -2,20 +2,29 @@
   <ds-space :class="{ notification: true, read: notification.read }" margin-bottom="x-small">
     <no-ssr>
       <ds-space margin-bottom="x-small">
-        <hc-user :user="post.author" :date-time="post.createdAt" :trunc="35" />
+        <hc-user
+          :user="post.author || comment.author"
+          :date-time="post.createdAt || comment.createdAt"
+          :trunc="35"
+        />
       </ds-space>
       <ds-text color="soft">
-        {{ $t('notifications.menu.mentioned') }}
+        {{ $t('notifications.menu.mentioned', { resource: post.id ? 'post' : 'comment' }) }}
```


```suggestion
        {{ $t('notifications.menu.mentioned', { resourceType }) }}
```

Reply:
> [<img alt="Tirokk" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/Tirokk) **Authored by [Tirokk](https://github.com/Tirokk)**
_<time datetime="2019-08-16T10:56:45Z" title="Friday, August 16th 2019, 12:56:45 pm +02:00">Aug 16, 2019</time>_

---


I did and it gets even more refactored in the next PR 🍰 Send out notifications on CreateComment #1018 …
For propper translations …